### PR TITLE
Selecting SmartProxy is not optional  in Global Reg procedure

### PR DESCRIPTION
**Pull request by swadeley**
Hello

See discussion https://github.com/theforeman/foreman-documentation/pull/386

Thank you

### CODE REVIEW

File: guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc [hunk 1]
```diff
@@ -56,7 +56,7 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
 
 . From the *Operating System* list, select the operating system of hosts that you want to register.
 
-. Optional: If you want to register hosts through {SmartProxy}, from the *{SmartProxy}* list, select the {SmartProxy} to register hosts to.
+. From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through {SmartProxy}.
```
Comment (spetrosi) [inline guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc:59]: Copying my comment from #386:
```suggestion
. From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
```
How about making it explicit like that?

Comment (swadeley) [reply guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc:59]: Fine with me.

Comment (spetrosi) [reply guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc:59]: You can either click **Commit suggestion**, or manually apply it and commit if you like. Then I will merge.

**Comment by swadeley** [comment]
Thank you

**Pull request merged**